test/integration/admin/api/account/proxy_configs_controller_test.rb

```diff
@@ -140,8 +117,44 @@ def setup
     assert_same_elements expected_proxy_config_ids_latest_version, response_proxy_config_ids
   end
 
+  test '#index with host param: it searches first for latest version of each proxy/service and then filters that result by host (the order of this matters)' do
+    service1 = FactoryBot.create(:simple_service, :with_default_backend_api, account: provider)
+    service2 = FactoryBot.create(:simple_service, :with_default_backend_api, account: provider)
```

Comment:
```suggestion
    service1, service2 = FactoryBot.create_list(:simple_service, 2, :with_default_backend_api, account: provider)
```

Reply:
I did it this way bcz it seemed to me by my chat with @josemigallas that it would be easier to understand 🤔